documentation/docs/tutorials/postgres-mcp.md

```diff
@@ -0,0 +1,205 @@
+    │  Add Extension 
+    │
+    ◇  What would you like to call this extension?
+    │  PostgreSQL
+    │
+    ◇  What command should be run?
+    │  npx -y @modelcontextprotocol/server-postgres postgresql://localhost/mydb
+    │     
+    ◇  Please set the timeout for this tool (in secs):
+    │  300
+    │    
+    // highlight-start
+    ◆  Would you like to add environment variables?
+    │  No 
+    // highlight-end
+    └  Added PostgreSQL extension
+  ```  
+
+  </TabItem>
+  <TabItem value="ui" label="Goose Desktop">
+  1. [Launch the installer](goose://extension?cmd=npx&arg=-y&arg=@modelcontextprotocol/server-postgres&id=postgres&name=PostgreSQL&description=PostgreSQL%20database%20integration&env=POSTGRES_URL%3DYour%20PostgreSQL%20connection%20URL)
+  2. Press `Yes` to confirm the installation
+  3. Enter your PostgreSQL connection URL in the format: `postgresql://username:password@hostname:5432/database`
+  4. Click `Save Configuration`
+  5. Scroll to the top and click `Exit` from the upper left corner
+  </TabItem>
+</Tabs>
+
+## Customizing Your Connection
+
+The PostgreSQL connection URL follows this format:
+```
+postgresql://username:password@hostname:5432/database
+```
+
+Where:
+- `username`: Your PostgreSQL user
+- `password`: Your PostgreSQL password
+- `hostname`: The host where PostgreSQL is running (e.g., localhost, IP address, or domain)
+- `5432`: The default PostgreSQL port (change if using a different port)
+- `database`: The name of your database
+
+Examples:
+- Local database: `postgresql://localhost/mydb`
+- Local with credentials: `postgresql://myuser:mypass@localhost/mydb`
+- Remote database: `postgresql://user:pass@db.example.com:5432/production`
+
+:::caution
+Never commit connection strings with credentials to version control! Use environment variables or secure configuration management.
+:::
+
+## Example Usage
+
+Let's see how to use Goose with the PostgreSQL extension to perform some common database operations.
+
+### Listing Tables
+
+#### Goose Prompt
+```
+Tell me which students in my database have a GPA above their grade's average.
+```
+
+#### Goose Output
+```
+Okay, here are the students with a GPA above their grade's average:
+
+*   **John Smith**, GPA: 3.85, Grade Level: 12, Grade Level Average GPA: 3.775
+*   **Emma Johnson**, GPA: 4.00, Grade Level: 11, Grade Level Average GPA: 3.90
+```
+
+### Querying Data
+
+#### Goose Prompt
+```
+Show me the top 5 orders of my store by value
+```
+
+#### Goose Output
+```
+I'll write and execute a query to find the highest value orders.
+
+Query results:
+| order_id | customer | total_value | order_date |
+|----------|----------|-------------|------------|
+| 1042     | ACME Inc | $5,280.00   | 2024-03-15 |
+| 1067     | TechCorp | $4,150.00   | 2024-03-18 |
+| 1039     | DataSys  | $3,900.00   | 2024-03-14 |
+| 1055     | InfoTech | $3,675.00   | 2024-03-16 |
+| 1071     | DevCo    | $3,450.00   | 2024-03-19 |
+
+```
```

Comment:
dont forget to update the goose output!

This was a great first tutorial. You were meticulous and caring about the developer 👏🏿

Reply:
Oh i see. You have multiple examples. Let's only use one. That seems to be the preferred format